client/src/components/AddCommentForm.jsx

```diff
@@ -35,27 +35,27 @@ const AddCommentForm = ({ questionId, answerId, setShowAddCommentForm }) => {
   });
 
   const changeHandler = (event) => {
+    const { value } = event.target;
+
     setComment((prevComment) => {
       return {
         ...prevComment,
         content: event.target.value,
         error:
-          event.target.value.length < 10 || event.target.value.length > 500
-            ? true
-            : false,
+          value.trim().length < 10 || value.trim().length > 500 ? true : false,
```

Comment:
Can we add https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/minlength the minlength and maxlength fields